app/views/partials/dependency_section.scala.html

```diff
@@ -16,38 +16,59 @@
 @(dependencies: Seq[Dependency])
 
+@defining(dependencies.filter(_.importBy.isDefined)) { transitiveDependencies =>
+    @for(dependency <- dependencies.filter(_.importBy.isEmpty).sortBy(_.name))  {
+      <li>
+        <div class="">
+            @dependencyRow(dependency)
+        </div>
 
-@for(dependency <- dependencies.sortBy(_.name))  {
-  <li>
-      <div class="@{getColourClass(dependency)}" id="@{dependency.name}">
-          <span class="col-xs-4 left-aligned">
-              <span id="@{dependency.name}-icon" class="@{getIconClass(dependency)}" title="@{getAltText(dependency)}"> </span>
-              @if(dependency.latestVersion.isDefined && !dependency.isExternal) {
-                  <a id="link-to-@{dependency.name}" href="/repositories/@{dependency.name}">@dependency.name</a>
-              } else {
-                  @dependency.name
-              }
-          </span>
-          <span id="@{dependency.name}-current-version" class="col-xs-3">@dependency.currentVersion.toString</span>
-          <span id="@{dependency.name}-latestVersion-version" class="col-xs-3">
-              <span class="glyphicon glyphicon-arrow-right small-glyphicon" style="padding-right: 10px;"> </span>
-              @dependency.latestVersion.map(_.toString).getOrElse("(not found)")
-          </span>
-          <span id="@{dependency.name}-bobbyrule" class="col-xs-2">
-              @dependency.versionState match {
-                  case Some(VersionState.BobbyRuleViolated(violation)) => { <a href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{violation.range}">See rule</a> }
-                  case Some(VersionState.BobbyRulePending (violation)) => { <a href="@{appRoutes.BobbyExplorerController.list()}#@{dependency.group}:@{dependency.name}:@{violation.range}">See rule</a> }
-                  case _                                               => { }
-              }
-          </span>
-      </div>
-  </li>
+        <div class="">
+            <ul>
+            @for(transitiveDependency <- transitiveDependencies.filter(_.importBy.contains(ImportedBy(dependency.name, dependency.group, dependency.currentVersion)))) {
+                <li>
+                @dependencyRow(transitiveDependency)
+                </li>
+            }
+            </ul>
+        </div>
+      </li>
+    }
+}
+
+@dependencyRow(dependency: Dependency) = {
+    <div class="@{getColourClass(dependency)} row" id="@{dependency.name}">
+        <span class="col-xs-4 left-aligned">
+            <span class="@if(dependency.importBy.isDefined){transitive}"></span>
+            <span id="@{dependency.name}-icon" class="@{getIconClass(dependency)}" title="@{getAltText(dependency)}"> </span>
+            @if(dependency.latestVersion.isDefined && !dependency.isExternal) {
+                <a id="link-to-@{dependency.name}" href="/repositories/@{dependency.name}">@dependency.name</a>
+            } else {
+                @dependency.name
```

Comment:
can we add the group name. Even for hmrc ones it's useful (e.g. hmrc-mongo is `uk.gov.hmrc.mongo`)